releases/net471/KnownIssues/479874-WPF Touch Stops Working After Prolonged Use of Popups.md

```diff
@@ -0,0 +1,12 @@
+## WPF Touch Stops Working After Rapid/Prolonged Use Of Popups
```

Comment:
Is this a compat issue that was fixed in 4.7.1. Or is it a break in 4.7 and 4.7.1?
In any case, it needs a `## Version Introduced` section, and if it was reverted in 4.7.1, a `##Version Reverted` section.